cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
+"""Manual VMware Provider tests"""
+import pytest
+
+pytestmark = []
+
+
+@pytest.mark.manual
+@pytest.mark.tier(3)
+def test_vmware_provider_filters():
+    """
+    N-3 filters for esx provider.
+    Example: ESXi 6.5 is the current new release.
+    So filters for 6.5 (n), 6.0 (n-1), 5.5 (n-2) at minimum.
+
+    Polarion:
+        assignee: kkulkarn
+        casecomponent: Provisioning
+        caseimportance: low
+        initialEstimate: 1/4h
```

Comment:
**Required** Need `testSteps` and `expectedResults` here and the below tests that are missing them. I would also like to see the doc blocks clearer on what is being tested here.

Reply:
Is it required for everything?

Reply:
Also does docblock translates to anything in polarion or just remains here in code?